doc/amo.md

```diff
@@ -50,6 +50,8 @@ Since Tridactyl aims to provide all the features Vimperator and Pentadactyl had,
     *   The URLs of websites you've visited previously can be suggested as arguments for `:tabopen` and similar commands.
 *   Access recently closed tabs:
     *   If you've accidentally closed a tab or window, Tridactyl will let you open it again with the `:undo` command which is bound to `u` by default.
+*   Monitor extension usage and manage themes:
```

Comment:
I *think* that this is the line produced by the `management` permission but I'm not certain; is there a resource that lays out the mapping, or do we guess by randomly sampling extensions and comparing their requested permissions against their manifests?